src/sut/dumblog/src/Dumblog/ZeroCopy/Sendfile.hs

```diff
@@ -1,22 +0,0 @@
-{-# LANGUAGE ForeignFunctionInterface #-}
-
-module Dumblog.ZeroCopy.Sendfile where
```

Comment:
Please remove this module from the .cabal file also.

Reply:
Fixed, thanks